robot/basestation/stream_capture.py

```diff
@@ -32,6 +33,7 @@ def start_recording_feed(stream_url):
     """
 
     global active_recordings
+    global video_filename
```

Comment:
Don't forget multiple videos can be recorded at the same time. You should not have a single global video file name. "active_recordings" contains all the video file names already so this is not useful

Reply:
Additionally you should modify "active_recordings" to contain a tuple (name, rotation). Or you could make a class and it would become a hashmap of that class. If you dont understand we can speak of it on Google Hangouts